Update docs for Python v0.5 (#1156)
inngest/website: content/blog/2025-06-23-python-v0.5.mdx

```

If you want to return a Pydantic object from a function, you can set the `output_type` parameter on the function:

```py
@client.create_function(
    fn_id="my-fn",
    output_type=User,
    trigger=inngest.TriggerEvent(event="my-event"),
)
async def my_fn(ctx: inngest.Context) -> None:
    return User(name="Alice")
```

## AI orchestration

The Python SDK now supports the same `step.ai.infer` AI orchestration as the TypeScript SDK. You can read more about AI orchestration in the [AI orchestration guide](/docs/features/inngest-functions/steps-workflows/step-ai-orchestration).

```py
import inngest
from inngest.experimental import ai

@inngest_client.create_function(
    fn_id="hello-world",
    trigger=inngest.TriggerEvent(event="say-hello"),
)
async def hello(ctx: inngest.Context) -> object:
    res = await ctx.step.ai.infer(
        "say-hello",
        adapter=ai.anthropic.Adapter(
            auth_key="sk-ant-000000",
            model="claude-3-5-sonnet-latest",
        ),
        body={
            "max_tokens": 1024,
            "messages": [{"role": "user", "content": "Hello, how are you?"}],
        },
    )
    return res["content"][0]["text"]
```

> Note that AI orchestration is still experimental. We feel that it's ready for production use, but the interface may change.

Reviewer: ```suggestion
Note that AI orchestration is still experimental. It is ready for production use, but the interface may change.
```
"we feel" feels kind of ambiguous and subjective, for something like this, we should use copy that is more clear.
